clj-refactor.el

```diff
@@ -1058,10 +1058,13 @@ word test in it and whether the file lives under the test/ directory."
     (when (and cljr-add-ns-to-blank-clj-files
                (cljr--clojure-ish-filename-p (buffer-file-name))
                (= (point-min) (point-max)))
-      (clojure-insert-ns-form)
-      (newline 2)
-      (when (cljr--in-tests-p)
-        (cljr--add-test-declarations)))))
+      ;; TODO: replace (if ...) with (cider-expected-ns) once dependable
```

Comment:
Oh I see.  Originally I was maintaining compatibility with cider 0.11, but now that we're past cljr 2.2.0 release, I guess we can assume `(fboundp #'cider-expected-ns)`.  Will make this change now...